src/Constants_Conversion.f90

```diff
@@ -75,27 +75,121 @@ MODULE Constants_Conversion
+REAL(SRK),PUBLIC,PARAMETER :: milli2centi=milli2base*base2centi
+REAL(SRK),PUBLIC,PARAMETER :: milli2micro=milli2base*base2micro
+REAL(SRK),PUBLIC,PARAMETER ::  milli2nano=milli2base*base2nano
+REAL(SRK),PUBLIC,PARAMETER ::  milli2pico=milli2base*base2pico
+REAL(SRK),PUBLIC,PARAMETER ::  micro2tera=micro2base*base2tera
+REAL(SRK),PUBLIC,PARAMETER ::  micro2giga=micro2base*base2giga
+REAL(SRK),PUBLIC,PARAMETER ::  micro2mega=micro2base*base2mega
+REAL(SRK),PUBLIC,PARAMETER ::  micro2kilo=micro2base*base2kilo
+REAL(SRK),PUBLIC,PARAMETER :: micro2centi=micro2base*base2centi
+REAL(SRK),PUBLIC,PARAMETER :: micro2milli=micro2base*base2milli
+REAL(SRK),PUBLIC,PARAMETER ::  micro2nano=micro2base*base2nano
+REAL(SRK),PUBLIC,PARAMETER ::  micro2pico=micro2base*base2pico
+REAL(SRK),PUBLIC,PARAMETER ::   nano2tera=nano2base*base2tera
+REAL(SRK),PUBLIC,PARAMETER ::   nano2giga=nano2base*base2giga
+REAL(SRK),PUBLIC,PARAMETER ::   nano2mega=nano2base*base2mega
+REAL(SRK),PUBLIC,PARAMETER ::   nano2kilo=nano2base*base2kilo
+REAL(SRK),PUBLIC,PARAMETER ::  nano2centi=nano2base*base2centi
+REAL(SRK),PUBLIC,PARAMETER ::  nano2milli=nano2base*base2milli
+REAL(SRK),PUBLIC,PARAMETER ::  nano2micro=nano2base*base2micro
+REAL(SRK),PUBLIC,PARAMETER ::   nano2pico=nano2base*base2pico
+REAL(SRK),PUBLIC,PARAMETER ::   pico2tera=pico2base*base2tera
+REAL(SRK),PUBLIC,PARAMETER ::   pico2giga=pico2base*base2giga
+REAL(SRK),PUBLIC,PARAMETER ::   pico2mega=pico2base*base2mega
+REAL(SRK),PUBLIC,PARAMETER ::   pico2kilo=pico2base*base2kilo
+REAL(SRK),PUBLIC,PARAMETER ::  pico2centi=pico2base*base2centi
+REAL(SRK),PUBLIC,PARAMETER ::  pico2milli=pico2base*base2milli
+REAL(SRK),PUBLIC,PARAMETER ::  pico2micro=pico2base*base2micro
+REAL(SRK),PUBLIC,PARAMETER ::   pico2nano=pico2base*base2nano
+
 !Pressure Conversions
 !> Conversion factor from MPa to psia
 REAL(SRK),PUBLIC,PARAMETER :: MPa2PSIA=145.0_SRK+377.0_SRK/9990.0_SRK
+!> Conversion factor from psia to MPa
+REAL(SRK),PUBLIC,PARAMETER :: PSIA2MPa=ONE/MPa2PSIA
 !> Conversion factor from bar to MPa
 REAL(SRK),PUBLIC,PARAMETER :: Bar2MPa=0.1_SRK
 !> Conversion factor from MPa to bar
 REAL(SRK),PUBLIC,PARAMETER :: MPa2Bar=ONE/Bar2MPa
 !> Conversion factor from MPa to Pa
-REAL(SRK),PUBLIC,PARAMETER :: MPa2Pa=1000000.0_SRK
+REAL(SRK),PUBLIC,PARAMETER :: MPa2Pa=mega2base
 !> Conversion factor from Pa to MPa
-REAL(SRK),PUBLIC,PARAMETER :: Pa2MPa=ONE/MPa2Pa
+REAL(SRK),PUBLIC,PARAMETER :: Pa2MPa=base2mega
 
 !Mass conversions
 !> Conversion factor from kg to g
-REAL(SRK),PUBLIC,PARAMETER :: kg2g=1000.0_SRK
+REAL(SRK),PUBLIC,PARAMETER :: kg2g=kilo2base
 !> Conversion factor from g to kg
-REAL(SRK),PUBLIC,PARAMETER :: g2kg=ONE/kg2g
+REAL(SRK),PUBLIC,PARAMETER :: g2kg=base2kilo
 !> Conversion factor from g to mg
-REAL(SRK),PUBLIC,PARAMETER :: g2mg=1000.0_SRK
+REAL(SRK),PUBLIC,PARAMETER :: g2mg=base2milli
 !> Conversion factor from mg to g
-REAL(SRK),PUBLIC,PARAMETER :: mg2g=ONE/g2mg
+REAL(SRK),PUBLIC,PARAMETER :: mg2g=milli2base
```

Comment:
One of these days (maybe as we wrap up the BWR STH stuff) I'll switch downstream uses over to the new ones and get rid of these redundant conversions.  Same for the length conversions further down.